js/scriptIndex.js

```diff
@@ -0,0 +1,16 @@
+window.onresize = function () {
+    console.log(window.outerWidth);
+    if (window.outerWidth < 800) {
+        let tamaño = window.outerWidth + "px " + "100%;";
+        let cadena = "background-size: ";
+        cadena += tamaño;

```

Comment:
Codacy has a fix for the issue: [Replace `········cadena·+=·tamaño;␍` with `····cadena·+=·tamaño;`](https://app.codacy.com/gh/Jeffer48/Consultarq/pullRequest?prid=9456241)
```suggestion
    cadena += tamaño;
```